docs/source/input_preparation.rst

```diff
@@ -0,0 +1,127 @@
+   Cases where this is observed, e.g. JNK1, there appears to be the possibility of an interaction between the perturbed ligand and the termini. Keeping the termini charged should retain the intended interaction.
+
+.. note::
+   Some tools, e.g. Pymol, are known to erroneously place caps out of order with the protein chain or TER cards between the protein chain and the cap. Please ensure that the caps appear in order (i.e. before or after the relevant termini residue), and that there are no TER cards between the termini residue and the cap.
+
+Non canonical amino acids (PTMs)
+================================
+
+Some of the input structures are known to have non-canonical amino acids, specifically TPO, PTR, or TYS residues. The OpenFE team has reviewed all such cases and the amino acid was found to be far from the binding site.
+
+As the OpenFE software cannot easily handle PTMs at this stage, **these residues should be modified back to their canonical alternative**. Tools such as Pymol offer the ability to mutate residues in this manner.
+
+Protonating inputs
+==================
+
+All files in the Schrodinger input files are considered to be in their intended protonation state. No additional protonation steps should be carried out.
+
+.. note::
+   1. Care should be taken that any processing steps do not alter the protonation state. For example, PyMol is known to change histidines from HID to HIE during capping.
+
+   2. To ensure that protonation states were not altered, please ensure that the prepared PDB file has the same number of protein atoms outside of capping groups.
+
+Setting residue names
+=====================
+
+Where possible, residues should be assigned PDB-compliant names.
+
+*Example 1: Waters named SPC (e.g. in the case of Thrombin in the JACS set), should be renamed to HOH.*
+
+*Example 2: Capping groups named NMA should be renamed to NME (e.g. in the case of PTP1B in the JACS set).*
+
+Fixing hydrogen atom names
+==========================
+
+In some cases, hydrogen names may need to be manually altered to match expected, i.e. PDB compliant, names.
+
+These exact cases can be difficult to identify, running the validation script (see below), will help identify these. Please reach out to the OpenFE team should you encounter any unknown hydrogen names.
+
+*Example 1: GLY termini hydrogens being named 3HA and HA instead of HA3 and HA2.*
+
+*Example 2: HIS (in the HID state) hydrogens being named 1HD, 2HD, and 1HE instead of HD1, HD2, and HE1.*
+
+Validating prepared files
+=========================
+
+To ensure that prepared files can be run using OpenFE, a short MD simulation validation script has been provided under
+`utils/input_validation.py <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/utils/onput_validation.py>`_.
+In an environment with OpenFE 1.0 installed, please run this script by calling:
+
+.. code-block:: python
+
+   # If you don’t have cofactors
+   python validate-inputs --pdb protein.pdb
```

Comment:
oops I forgot there was already an assigned name, can you switch these over to `input_validation` please? Sorry about that.